plane_spotter/geolocator.py

```diff
@@ -1,15 +1,87 @@
 from typing import Any
 
+import csv
+import haversine
+
+AIRPORT_TYPE_BLACKLIST = set(('balloonport', 'closed', 'heliport', 'seaplane_base'))
+
 class Geolocator:
   ''' Translates coordinates to the closest airport. '''
 
   def __init__(self, airport_code_file: str='plane_spotter/data/airport-codes.csv') -> 'Geolocator':
```

Comment:
I would actually make `airport_code_file` a `pathlib.Path` type now that I think of it.

Reply:
So much to relearn in Python...